crates/nu-command/tests/commands/open.rs

```diff
@@ -65,6 +96,17 @@ fn parses_bson() {
     assert_eq!(actual.out, "hello");
 }
 
+#[cfg(feature = "bson")]
+#[test]
+fn parses_bson() {
+    let actual = nu!(
+        cwd: "tests/fixtures/formats",
+        "open sample.bson | get root | select 0 | get b"
```

Comment:
Not sure how this test case relates to the fix?

Reply:
ah that's an artifact from when I thought I needed to duplicate all the tests. I'll remove it